package.json

```diff
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/Eppo-exp/node-server-sdk#readme",
   "dependencies": {
-    "@eppo/js-client-sdk-common": "^4.6.3"
+    "@eppo/js-client-sdk-common": "4.8.0-alpha.1"
```

Comment:
Will publish after this https://github.com/Eppo-exp/js-sdk-common/pull/190 gets reviewed, and then all the automatic PR checks should pass (they pass locally)